src/NUnitFramework/testdata/SetUpFixtureData.cs

```diff
@@ -610,3 +610,154 @@ public void Test()
     }
 }
 #endregion
+
+#region SomeFixtureWithExceptions
+[TestFixture]
+public class SomeBaseFixtureExceptionInSetUp
+{
+	private readonly bool exceptionInFixtureSetUp;
```

Comment:
The coding standard is 4 spaces and tabs converted to spaces. You are using tab characters. Can you please convert to spaces? It will make working with future diffs to this code easier to read.